README.md

```diff
@@ -1,6 +1,45 @@
-# hypermodules
+<h1 align="center">✨<a href="https://github.com/hypermodules">hypermodules</a>⚡️</h1>
```

Comment:
ok ill change it.  It looks pretty scrappy either way.  I might try to make it look more professional at some point.